player/stream-test/js/script.js

```diff
@@ -2,19 +2,33 @@ window.onload = function getURLParams() {
   var manifestParam = getParamsQueryString('manifest');
   var streamFormatParam = getParamsQueryString('format');
   if (manifestParam && streamFormatParam) {
-    document.getElementById('stream_format').value = streamFormatParam;
-    document.getElementById('manifest').value = manifestParam;
+    document.getElementsByName('stream-format').forEach(function (element) {
+      if (element.value === streamFormatParam) {
+        element.checked = true;
+      }
+    })
+    document.getElementById('manifest-input').value = manifestParam;
+
+    loadPlayerFromControls();
+  }
+  else {
+    setupPlayer('dash', config.source.dash);
   }
 };
 
-var manifest = document.getElementById('manifest');
-manifest.onkeyup = handleKeyPress;
+var initialTimestamp, bufferChart, bitrateChart;
+var updateCount = 0;
+
+var loadManifestButton = document.getElementById('manifest-load');
+loadManifestButton.onclick = loadPlayerFromControls;
```

Comment:
[`addEventListener`](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener) is better, since it handlers can't be overwritten just by assigning a variable and it has many additional options.